packages/react-component-playground/src/components/component-playground.jsx

```diff
@@ -11,8 +11,14 @@ var _ = require('lodash'),
     isSerializable = require('../lib/is-serializable.js').isSerializable,
     SplitPane = require('ubervu-react-split-pane'),
     localStorageLib = require('../lib/local-storage.js'),
+    CodeMirror = require('react-codemirror'),
     fuzzaldrinPlus = require('fuzzaldrin-plus');
 
+require('codemirror/lib/codemirror.css');
+require('codemirror/addon/fold/foldgutter.css');
+require('codemirror/theme/solarized.css');
```

Comment:
Let's keep the `react-code-mirror` with minimum changes, besides the `react-dom-polyfill` integration, to be able to submit a PR upstream. I'll take a closer look on what you added in that fork as well.